### PR TITLE
Fix copyright year

### PR DESCRIPTION
**Pull request by oalders**

### CODE REVIEW

**Comment by karenetheridge** [comment]
why set the ending year? You're just setting yourself up for having to update it every January :)

File: dist.ini [hunk 1]
```diff
@@ -3,7 +3,7 @@ author  = Gisle Aas <gisle@activestate.com>
 license = Perl_5
 main_module = lib/LWP/Protocol/https.pm
 copyright_holder = Gisle Aas
-copyright_year   = 2017
+copyright_year   = 1997-2019
```
Comment (genio) [inline dist.ini]: As @karenetheridge pointed out, this should probably just be 1997. I'm not sure why we ever did the year range thing (likely my fault).

**Comment by oalders** [comment]
I don't know why I started doing the year ranges.  I'll back out of that change (and that habit)!

**Comment by oalders** [comment]
Thanks @karenetheridge and @genio!

**Pull request merged**